app/student/forms.py

```diff
@@ -0,0 +1,43 @@
+from flask import url_for
+from flask_wtf import Form
+from wtforms import ValidationError
+from wtforms.fields import (StringField, IntegerField, SubmitField)
+from wtforms.validators import Email, EqualTo, InputRequired, Length
+
+class EditCommonAppForm(Form):
```

Comment:
nit. can we change this to EditCommonAppEssayForm?

Reply:
Also this keeps it consistent w the function edit_common_app_essay() defined below